docs/release-notes/cli-requires-initial-password.rst

```diff
@@ -0,0 +1,9 @@
+:orphan:
+
+Breaking Changes
+
+-  Master: On new deployments, the service will log an error and abort startup if no
+   ``initialUserPassword`` is found in the configuration.
+
+To ensure users can still rely on reasonable default settings with CLI commands like ``det deploy
```

Comment:
Same as another comment about calling it "default".  Maybe something like initialization settings, bootstrap settings, etc?